extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.js

```diff
@@ -70,6 +72,27 @@ const ALLOWED_AD_TYPES = map({
   'doubleclick': true,
 });
 
+/** @enum {string} */
+const EVENTS = {
+  AD_REQUESTED: 'story-ad-request',
+  AD_LOADED: 'story-ad-load',
+  AD_INSERTED: 'story-ad-insert',
+  AD_VIEWED: 'story-ad-view',
+  AD_CLICKED: 'story-ad-click',
+  AD_EXITED: 'story-ad-exit',
+  AD_DESTROYED: 'story-ad-destroy',
+};
+
+/** @enum {string} */
+const VARS = {
+  AD_REQUESTED: 'requestTime',
+  AD_LOADED: 'loadTime',
```

Comment:
please add comments to each item, especially when is "loaded", "inserted".

Reply:
done.